consvc_shepherd/settings.py

```diff
@@ -140,3 +141,16 @@
         },
     },
 }
+# Sentry Setup
+SENTRY_SDK = env("SENTRY_SDK", default=None)
+SENTRY_TRACE_SAMPLE_RATE = env("SENTRY_TRACE_SAMPLE_RATE", default=1.0)
+sentry_sdk.init(
+    dsn=SENTRY_SDK,
+    integrations=[DjangoIntegration()],
```

Comment:
Need to specify the environment at the very minimum. 
```suggestion
    integrations=[DjangoIntegration()],
    environment=SENTRY_ENV,
```